File: template/java/solve.java
import my_libraries.Cp;

public class solve {
    static void solve() throws Exception {
        Cp.println("Hello, World!");
    }

    /*   /\_/\
        (= ._.)
        / >  \>
    */

    public static void main(String[] args) throws Exception {
        Cp.Timer timer = new Cp.Timer();

        int tc = 1; // tc = Cp.fs.nextInt();
        while (tc-- > 0) {
            solve();
        }

        Cp.flush();
        Cp.out("\n");
        Cp.out(String.format(java.util.Locale.US, "%.6fs", timer.elapsed()));
    }
}

File: template/python/solve.py
from my_libraries.cp import *


def solve():
    print_("Hello, World!")


#   /\_/\
#  (= ._.)
#  / >  \>


def main():
    timer = Timer()

    tc = 1  # tc = ni()
    for _ in range(tc):
        solve()

    flush()
    timer_out(timer.elapsed())


if __name__ == "__main__":
    main()
